src/control/server/storage/faults.go

```diff
@@ -14,30 +14,47 @@ import (
 	"github.com/daos-stack/daos/src/control/fault/code"
 )
 
-// FaultScmNotInterleaved represents an error where PMem regions exist in non-interleaved
+// FaultScmNotInterleaved creates a fault for the case where the PMem region is in non-interleaved
 // mode, this is unsupported.
-var FaultScmNotInterleaved = storageFault(code.ScmNotInterleaved,
-	"PMem regions are in AppDirect non-interleaved mode",
-	"Rerun the command first with the --reset option and then without to recreate the "+
-		"PMem regions in the recommended mode")
+func FaultScmNotInterleaved(sockID uint) *fault.Fault {
+	return storageFault(
+		code.ScmBadRegion,
+		fmt.Sprintf("PMem region on socket %d is in non-interleaved mode which is unsupported", sockID),
+		"Remove and recreate region in AppDirect interleaved mode")
```

Comment:
done, @kjacque we only remove the old goals so that when the command is invoked any stale goals won't interfere with any new goals that are set as part of the state change. The process of removing regions requires new goals to be set (after any stale ones have been cleared) before a reboot.